controllers/cluster_controller.go

```diff
@@ -707,15 +698,22 @@ func GetJSONHash(object interface{}) (string, error) {
 	return hex.EncodeToString(specHash), nil
 }
 
-// GetDynamicConfHash gets a hash of the data from the config map holding the
+// getDynamicConfHash gets a hash of the data from the config map holding the
 // cluster's dynamic conf.
 //
 // This will omit keys that we do not expect the pods to reference.
-func GetDynamicConfHash(configMap *corev1.ConfigMap) (string, error) {
-	var data = make(map[string]string, len(configMap.Data))
-	for key, value := range configMap.Data {
-		data[key] = value
+func getDynamicConfHash(configMap *corev1.ConfigMap, pClass fdbtypes.ProcessClass, serversPerPod int) (string, error) {
```

Comment:
Are we sure that there aren't other entries in the config map that are relevant to the processes? For instance, I think  we include the CA file in the config map.

Reply:
Though that may raise further questions about how we're rolling out changes like that and what the purpose if including them in the hash is.

Reply:
Yes, the `ca-file` is part of the configMap. I can add that and I guess it makes sense from a rollout perspective to include it.